app/experimenter/nimbus-ui/package.json

```diff
@@ -51,7 +51,6 @@
     "react-json-pretty": "^2.2.0",
     "react-markdown": "^5.0.3",
     "react-scripts": "3.4.0",
-    "react-scrollspy": "^3.4.3",
```

Comment:
Oh good, finally getting rid of this.  But also try not to fold too many things into a single pr you never know if you'll have to revert it and then lose multiple changes 🙏

Reply:
Well, this folded into [this bit here](https://github.com/mozilla/experimenter/pull/6620/files#diff-89aaf3f025dcf630551a25637e625f6ab63a8f9b1967abed0a1443de2ff7c8d1L159-L225) where I was already removing a section and removing ScrollSpy simplified the whole thing.

Reply:
Yeah if it makes sense together then is good 👍